core/unit_test/CMakeLists.txt

```diff
@@ -546,6 +546,7 @@ IF(KOKKOS_ENABLE_OPENACC AND KOKKOS_CXX_COMPILER_ID STREQUAL NVHPC)
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewAPI_f.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewMapping_b.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewResize.cpp
+    ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_BitManipulationBuiltins.cpp
```

Comment:
If it's not related to this pull request, I would prefer to discuss this change in a different pull request so that this one doesn't get blocked by it.

Reply:
Undo and propose elsewhere as necessary.

Reply:
Removed.

Reply:
This failed unit test problem is handled by issue #6088.